Makefile

```diff
@@ -56,15 +64,7 @@ generate:
 
 .PHONY: compile
 compile:
-	go build -o=gitea-operator ./cmd/manager/main.go
-
-.PHONY: packr
-packr:
-	packr
-
-.PHONY: packr-clean
-packr-clean:
-	packr clean
```

Comment:
Did you intend to remove these packr targets?

Reply:
nvm, i see this now https://github.com/integr8ly/gitea-operator/pull/22. I guess this was copied from elsewhere and had these left in it.